packages/components/src/Actions/utils.js

```diff
@@ -0,0 +1,27 @@
+import omit from 'lodash/omit';
+
+const IGNORED_PROPS = [
+	'bsStyle',
+	'inProgress',
+	'disabled',
+	'hideLabel',
+	'link',
+	'onMouseDown',
+	'tooltipPlacement',
+	'tooltip',
+	'tooltipLabel',
+	'available',
+];
+
+export default function getRClick(action) {
+	const { label, model, onClick, ...rest } = omit(action, IGNORED_PROPS);
```

Comment:
Why don't you do the props restriction prior to calling `getRClick`?
It look like a lot of work to do it on every user click.